features/extract_command.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Extracting theme contents to source
```

Comment:
if by **test**, you meant minitest module, I encountered some roadblocks while running it locally and decided to settle for a cucumber feature.